apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 
 <LiveExample src='ProgressLinear.main.jsx' truncate={false}>
   <AllExamples.ProgressLinearMainExample client:load />
 </LiveExample>
 
-### Props
+<LiveExample src='ProgressRadial.main.jsx' truncate={false}>
+  <AllExamples.ProgressRadialMainExample client:load />
+</LiveExample>
 
-<PropsTable path='@itwin/itwinui-react/esm/core/ProgressIndicators/ProgressLinear.d.ts' />
+### Value
 
-## Progress radial
+To specify a value for indeterminate indicator, pass a number into the `value` prop. This prop is available for both linear and radial indicators. If the `indeterminate` prop is set to `true`, this value will be ignored. This value must be inclusively between 0 and 100.
 
-<Placeholder componentPageName='progressradial--determinate' />
+<LiveExample src='ProgressLinear.value.jsx' truncate={false}>
+  <AllExamples.ProgressLinearValueExample client:load />
+</LiveExample>
 
-<LiveExample src='ProgressRadial.main.jsx' truncate={false}>
-  <AllExamples.ProgressRadialMainExample client:load />
+<LiveExample src='ProgressRadial.value.jsx' truncate={false}>
+  <AllExamples.ProgressRadialValueExample client:load />
+</LiveExample>
+
+Exclusively for linear indicators, the `isAnimated` flag is helpful when applying animation to the indicator as the value changes.
+
+<LiveExample src='ProgressLinear.isAnimated.jsx'>
+  <AllExamples.ProgressLinearAnimatedExample client:load />
+</LiveExample>
+
+### Status
+
+Three available statuses for both linear and radial progress indicators are:
+
+- `"positive"`
+- `"negative"`
+- `"warning"`
+
+<LiveExample src='ProgressLinear.status.jsx'>
+  <AllExamples.ProgressLinearStatusExample client:load />
+</LiveExample>
+
+Each radial indicator with status is accompanied by an icon, providing a visual cue to inform the current progress state better. The icon can be replaced by a [content](#content).
+
+<LiveExample src='ProgressRadial.status.jsx'>
+  <AllExamples.ProgressRadialStatusExample client:load />
+</LiveExample>
+
+### Label
+
+Indicator labels can be passed into the `labels` prop as an array. If one label is passed, it will be centered. If there are two labels, each is pushed to one end of the progress indicator. This prop is only available for linear indicators.
+
+<LiveExample src='ProgressLinear.label.jsx'>
+  <AllExamples.ProgressLinearLabelExample client:load />
+</LiveExample>
+
+### Size
+
+Four size options available are:
+
+- `"x-small"`
+- `"small"`
+- `"medium"` (default)
+- `"large"`
+
+This prop is only available for radial indicators.
+
+<LiveExample src='ProgressRadial.size.jsx'>
+  <AllExamples.ProgressRadialSizeExample client:load />
+</LiveExample>
+
+### Content
+
+The radial progress indicator can be wrapped around JSX elements, such as icons or texts. This content helps provide flexibility to enhance the progress display with additional visual appearance. Moreover, indicators of `"x-small"` size will not display its content. This prop is only available for radial indicators.
```

Comment:
The "moreover" sentence could be moved below the demo and turned into a "Note:", so that it doesn't distract from the main point here.

Reply:
I have moved it to a "Note".

Reply:
I actually meant it should be _below the demo_ (at the end of the section). In my mind, the demo should come first because it's not really affected by the note.

Reply:
I have moved this below the demo.